nmosauth/auth_server/token_generator.py

```diff
@@ -20,7 +20,8 @@
 from .oauth2 import authorization
 from .constants import NMOSAUTH_DIR, PRIVKEY_FILE
 
-ALLOWED_SCOPES = ["registration", "node", "query", "connection"]
+SCOPES_SUPPORTED = ["registration", "node", "query", "connection"]
```

Comment:
Whilst we haven't done explicit testing of this yet, it might be worth including scopes for all of the current NMOS APIs

Reply:
Currently the auth server only supports the ones listed. If we included all the others then you'd end up with keys appearing in the x-nmos-api claim that are subsequently empty, which isn't permitted in the spec